libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateRangePickerInputField.tsx

```diff
@@ -17,18 +19,19 @@ interface IDateRangePickerInputFieldProps extends DayPickerInputProps {
     classNameCalendar?: string;
 }
 
-function formatDate(date: Date, format: string, locale: string): string {
-    return moment(date).locale(locale).format(format);
+function formatDate(date: Date, dateFormat: string): string {
+    return format(date, dateFormat);
 }
 
-function parseDate(str: string, format: string, locale: string): Date | void {
-    const result = moment(str, format, locale, true);
-
-    if (result.isValid()) {
-        return result.toDate();
+function parseDate(str: string, dateFormat: string): Date | void {
```

Comment:
Should probably return `undefined` if parsing fails. Return type of `anything | void` is strange.